src/redux/data.js

```diff
@@ -232,8 +232,21 @@ export const addressAssetsReceived = (
 
   const liquidityTokens = remove(
     assets,
-    asset => asset.asset.type === 'uniswap'
+    asset =>
+      asset?.asset?.type === 'uniswap' ||
+      toLower(asset?.asset.name).indexOf('uniswap') !== -1
   );
+
+  // Remove spammy tokens
+  remove(assets, asset =>
```

Comment:
These shitcoins and the compound tokens should not be included in assets (given they should be filtered out on line 221 with "compound" and "trash" types - were you still seeing these included?